### ISSUE TITLE
The issue of handling LQ images in the code for the restoration task.

### ISSUE THREAD

**Issue by fn6767** [Algolzw/image-restoration-sde#130]
Hello, I have a question about the restoration task. Is the LQ image the one with missing regions in the mask area? If so, why does the `mask_to` function return `gt * mask + (1 - mask)`? What does `gt * mask + (1 - mask)` mean? 
[(https://github.com/Algolzw/image-restoration-sde/blob/2598d73cde3a1e2cf78c4ba4923414e1f4b90393/codes/utils/deg_utils.py#L34)]

**Comment by fn6767** [comment]
[https://github.com/Algolzw/image-restoration-sde/blob/2598d73cde3a1e2cf78c4ba4923414e1f4b90393/codes/utils/deg_utils.py#L34C5-L34C39](url)

def mask_to(tensor, mask_root='data/datasets/gt_keep_masks/genhalf', mask_id=-1, n=100):
    batch = tensor.shape[0]
    if mask_id < 0:
        mask_id = np.random.randint(0, n, batch)
        masks = []
        for i in range(batch):
            masks.append(cv2.imread(os.path.join(mask_root, f'{mask_id[i]:06d}.png'))[None, ...] / 255.)
        mask = np.concatenate(masks, axis=0)
    else:
        mask = cv2.imread(os.path.join(mask_root, f'{mask_id:06d}.png'))[None, ...] / 255.
    mask = torch.tensor(mask).permute(0, 3, 1, 2).float()
    # for images are clipped or scaled
    mask = F.interpolate(mask, size=tensor.shape[2:], mode='nearest')
    masked_tensor = mask * tensor
    **return masked_tensor + (1. - mask)**

**Comment by Algolzw** [comment]
Hello, we use the code `gt * mask + (1 - mask)` for masking since we want the masked areas to be the white color (means values are 1 instead of 0). You can feel free to change it if you want the masked area to be black.
